### PR TITLE
Add new setting to show / hide articles thumbnails

### PR DESCRIPTION
**Pull request by nicosomb**
| Q             | A
| ------------- | ---
| Bug fix?      |no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | no
| Documentation | no
| Translation   | yes
| CHANGELOG.md  | no
| License       | MIT

Should fix https://github.com/wallabag/wallabag/issues/6346. 

@j0k3r @Kdecherf about tests on this PR, what is the best way? A functional test to check if the element is in the DOM or not? 

### Card view

Capture d’écran 2023-06-13 à 14 45 53

### List view 

Capture d’écran 2023-06-13 à 14 45 46

### Card view

We lose the tags here, but we have to make choice. 

Capture d’écran 2023-06-13 à 14 45 31

### List view 

Capture d’écran 2023-06-13 à 14 45 38

## The configuration screen with a checkbox:

Capture d’écran 2023-06-13 à 15 00 08

This checkbox needs some love: @Simounet, maybe you can help me here?

### CODE REVIEW

File: src/Wallabag/CoreBundle/Entity/Config.php [hunk 1]
```diff
@@ -117,6 +117,15 @@ class Config
      */
     private $listMode;
 
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="display_thumbnails", type="integer", nullable=true)
```
Comment (j0k3r) [inline src/Wallabag/CoreBundle/Entity/Config.php:123]:
```suggestion
     * @ORM\Column(name="display_thumbnails", type="integer", nullable=true, options={"default" = 1})
```

**Review by j0k3r — COMMENTED** [review]
Maybe you can just check if there is span with the class `preview`?

**Comment by Simounet** [comment]
Here is my proposal:
wallabag-settings

I fixed the incorrect width for the mark as read default action. Wanted to fix the labels height on mobile, but, come on materialize, position absolute on labels? You're drunk… Such a bad UI idea. Will be for another day.

**Comment by j0k3r** [comment]
Conflicts :)

**Review by j0k3r — APPROVED** [review]

**Pull request merged**